iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/persistence/pipe/PipeTaskInfo.java

```diff
@@ -80,12 +84,51 @@ private void releaseReadLock() {
 
   private void acquireWriteLock() {
     pipeMetaKeeper.acquireWriteLock();
+    pipeTaskInfoVersion.increaseLatestVersion();
   }
 
   private void releaseWriteLock() {
     pipeMetaKeeper.releaseWriteLock();
   }
 
+  /////////////////////////////// Version ///////////////////////////////
+
+  private class PipeTaskInfoVersion {
```

Comment:
It is recommended to remove this class from PipeTaskInfo, as Info records content that needs to be persisted to the consensus layer. PipeTaskInfoVersion can be placed in the PipeTaskCoordinator class (or its package)